packages/ui/src/components/Visualization/Canvas/canvas.service.ts

```diff
@@ -38,6 +40,19 @@ export class CanvasService {
     newController.registerLayoutFactory(this.baselineLayoutFactory);
     newController.registerComponentFactory(this.baselineComponentFactory);
 
+    newController.addEventListener(GRAPH_LAYOUT_END_EVENT, () => {
+      newController.getGraph().fit(80);
+    });
+
```

Comment:
If we move this event listener to the `Canvas` component we have the chance to provide a cleanup function in order to remove it.

```suggestion
```